server/src/jobs/domainesMetiers/updateDomainesMetiers.ts

```diff
@@ -96,21 +99,32 @@ export default async function (optionalFileName?: string) {
         // cas de la ligne sur laquelle se trouve le nom du métier qui va marquer l'insertion d'une ligne dans la db
         step = 1
 
-        const paramsDomaineMetier = {
+        const paramsDomaineMetier: IDomainesMetiers = {
           domaine: domaine,
+          domaine_sans_accent_computed: removeAccents(domaine),
           sous_domaine: metier,
+          sous_domaine_sans_accent_computed: removeAccents(metier),
           mots_clefs_specifiques: [...new Set(motsClefsSpecifiques)].join(", "),
+          mots_clefs_specifiques_sans_accent_computed: removeAccents([...new Set(motsClefsSpecifiques)].join(", ")),
```

Comment:
```suggestion
          mots_clefs_specifiques_sans_accent_computed: [...new Set(motsClefsSpecifiques.map(removeAccents))].join(", "),
```
attrape aussi les cas où les mots sans accent sont les mêmes (â et à  => a)

Reply:
a voir aussi pour les autres champs si c'est intéressant